fdbclient/SystemData.h

```diff
@@ -223,6 +223,13 @@ extern const KeyRef excludedServersVersionKey; // The value of this key shall be
 const AddressExclusion decodeExcludedServersKey(KeyRef const& key); // where key.startsWith(excludedServersPrefix)
 std::string encodeExcludedServersKey(AddressExclusion const&);
 
+extern const KeyRef excludedLocalityPrefix;
```

Comment:
With C++17 there are now `inline` variables, so I would recommend using:

```
inline const KeyRef excludedLocalityPrefix = ...;
```

Reply:
They are lot other extern variables in this file. I will add a to-do and will address later. Thanks!